healthcheck

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ $# -ne 2 ]; then
+    echo "Usage: $0 HOST PORT
+
+Healthcheck for the RT docker container."
+    exit 1
+fi
+
+host="$1"
+port="$2"
+
+resp=$(curl -s -d @- -H "Host: rt.ocf.berkeley.edu" \
+       "http://$host:$port/REST/1.0/ticket/1/view" <<EOF
+user=create&pass=$(cat /opt/share/secrets/rt/healthcheck.passwd)
```

Comment:
Should we create a separate user for this check?

Reply:
`create` isn't in LDAP or kerberos, it's just a local RT user, right? In that case, I think this is fine.

Reply:
I think using create for this is fine. Maybe it's not ideal, but I think it's better than adding a whole new user for the health check.